andela/titan-dev-ah-frontend
# #163518649 Comment on article

#### What does this PR do?
- Comment on article
#### Description of Task to be completed?
- Enable comment on an article
#### How should this be manually tested?
- clone the repos and check out the branch `ft-comment-on-article-163518649`
- clone the repos and check out the branch `bg-fix-comments-165964990`
- Run `yarn` to install all dependencies on both branch
- Run `yarn test` to run tests, they should all pass on both branch
- Start your backend server on the address `localhost:3000/` by issuing `yarn start` command
- Create some articles using client tool like the postman 
- Grab slug of one of the articles you created. 
- Run `yarn start` to start your server
- Navigate to `localhost:8080/articles/:{slug}`
- Comment on an article you should see comment under the article.
#### Any background context you want to provide?
- None
#### What are the relevant pivotal tracker stories?
# 163518649
